26-register_registration/creator_navigates_to_description_tab.feature

```diff
@@ -90,3 +90,36 @@ Feature: Creator navigates to Description tab
     Given Creator begins Wizard registration and navigates to Description tab
     When the user opens the Project wizard Dialog for creating a new Project
     Then User opens the Project Wizard to register a new Project
+
+  @test
+  Scenario: Creator adds funding
+    Given Creator begins Wizard registration and navigates to Description tab
+    When they uses the option to add funding
+    Then they see an option to add a funding source
+    And they see an option to cancel the funding source
+
+  @test
+  Scenario: Creator adds funding from NFR
+    Given Creator begins Wizard registration and navigates to Description tab
+    And they add funding
+    When they select NFR as a funding source
+    Then they can search for NFR Project
+    
+  @test
+  Scenario: Creator adds funding from a NFR Project
+    Given Creator begins Wizard registration and navigates to Description tab
+    And they add funding
+    And they select NFR as a funding source
+    When they select a NFR Project
+    Then they can register the sum for the funding
```

Comment:
Man skal vel ikke kunne oppgi sum når man legger til NFR-prosjekt, da den infoen ligger hos NFR? Det er i hvert fall slik det er implementert nå, basert på skisser

Reply:
Ok, scenario oppdatert.

Reply:
Sum will never be returned from NFR as of now. My impression from @garshol is that is is an optional free text field for all fundings independent of source.

Reply:
Når vi har eit NFR tilskudd (den nye termen for Grant), så finnes det ein landingpage som gir detaljer, som sum. Derfor er det ikkje nødvendig å lagre dette i NVA. Dette vil også gjelde for tilskudd som kjem fra EU/CORDIS, dei har DOI.

Når nokon påstår å ha fått eit tilskudd fra nokon (Les:Historielaget i Huttameitu) så finnes der ingen landingpage som kan gi informasjon - derfor åpner vi for å kunne lagre summen.

Uansett; sum er ikkje viktig.
Det vesentlige er om prosjektet har eit tilskudd, og kven det er som gir det.

Reply:
Der er stor motvilje mot at prosjektregisteret skal ta "form og farge" som eit prosjektoppfølgings-verktøy. Vi skal ikkje dekke nokon økonomiske prosjekt funksjon. Vi skal vise kven det er som finansierer forskningen, detaljer er unødvendig (dei finnes andre plasser, i institusjonene sine prosjektregnskap)